modules/core/src/main/java/org/apache/ignite/internal/processors/cache/local/atomic/GridLocalAtomicCache.java

```diff
@@ -650,6 +652,8 @@ public GridLocalAtomicCache(GridCacheContext<K, V> ctx) {
         if (keyCheck)
             validateCacheKeys(keys);
 
+        checkKeysOrdered(keys, "Invoke All");
```

Comment:
Here should be `"Invoke All Sync"` operation.

Reply:
Do you mean "Async"?? Others also

Reply:
Yes, I do.

Reply:
I have dropped "Async" indication altogether, it will be misreported often.